addons/digest/tests/test_digest.py

```diff
@@ -0,0 +1,75 @@
+from unittest.mock import patch
+import random
+
+from dateutil.relativedelta import relativedelta
+from lxml import html
+
+from odoo.addons.mail.models.mail_mail import MailMail
+
+from odoo import fields
+from odoo.tests import TransactionCase
+
+
+class TestDigestStuff(TransactionCase):
+    def test_digest_numbers(self):
+        self._setup_messages()
+
+        d = self.env['digest.digest'].create({'name': "My Digest", 'kpi_mail_message_total': True})
+
+        # subscribe the demo user so at least one mail gets sent
+        demo = self.env.ref('base.user_demo')
```

Comment:
Make a specific user for tests, better than relying on existing data (there are standard users in mail common, a standard employee for example).